### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by jarno-k**
Improved documentation for L filter, LCL filter and DC bus models.

### CODE REVIEW

File: docs/source/model/filter_and_grid.rst [hunk 1]
```diff
@@ -39,10 +39,80 @@ function :func:`gritulator.complex2abc`.
 L Filter 
 --------
 
-This section will be extended in the future.
+A dynamic model for an inductive L filter and inductive-resistive grid impedance 
+is provided in the package :mod:`gritulator.model._grid_filter`. The model is
```
Comment (rayanmour) [inline docs/source/model/filter_and_grid.rst:43]: Could we add a diagram of the L filter (and LCL filter in the next section) similarly to motulator documentation for "Electric Machines"?

File: docs/source/model/filter_and_grid.rst [hunk 2]
```diff
@@ -39,10 +39,80 @@ function :func:`gritulator.complex2abc`.
...
+   :label: L_filt_PCC_voltage
 
 
 LCL Filter
 ----------
 
-This section will be extended in the future.
+A dynamic model for an inductive-capacitive-inductive (LCL) filter and 
+inductive-resistive grid impedance is also provided in the package 
+:mod:`gritulator.model._grid_filter`. The model is implemented in stationary
```
Comment (rayanmour) [inline docs/source/model/filter_and_grid.rst:85]: Could we add LCL filter diagram? (see previous comment)? Then the PCC could somehow be more highlighted if needed.

File: docs/source/model/system.rst [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Grid-converter System
+Grid-Converter System
 =====================
 
 Grid-converter systems are sampled-data systems, consisting of continuous-time
```
Comment (rayanmour) [inline docs/source/model/system.rst]: Grid converters (to avoid systems twice?)

File: gritulator/model/dc_bus/_dc_dyn_model.py [hunk 4]
```diff
@@ -108,9 +108,10 @@ def f(self, t, x):
         e_gs = self.grid_model.voltages(t)
         q = self.converter.q
         i_g_abc = complex2abc(i_gs)
+        i_dc = self.dc_model.dc_current(i_g_abc, q) # DC-current
         # State derivatives
         rl_f = self.grid_filter.f(i_gs, u_cs, e_gs)
-        dc_f = self.dc_model.f(t, u_dc, i_g_abc, q)
+        dc_f = self.dc_model.f(t, u_dc, i_dc)
```
Comment (rayanmour) [inline gritulator/model/dc_bus/_dc_dyn_model.py:114]: Nice simplification

File: docs/source/model/filter_and_grid.rst (hunk 1 above)
Comment (jarno-k) [reply docs/source/model/filter_and_grid.rst:43]: Yes, done.

File: docs/source/model/system.rst (hunk 3 above)
Comment (jarno-k) [reply docs/source/model/system.rst]: Yes, done.

File: docs/source/model/filter_and_grid.rst (hunk 2 above)
Comment (jarno-k) [reply docs/source/model/filter_and_grid.rst:85]: Yes, done.

**Pull request merged**